packages/contracts/contracts/vesting/Vesting.sol

```diff
@@ -64,32 +69,32 @@ contract Vesting is OwnablePausableUpgradeable, ReentrancyGuardUpgradeable, IVes
             amountReleased: 0,
             nextReleaseStep: 0
         });
-        vestingSchedules[beneficiary][orderId] = newVesting;
+        vestingSchedules[newVestingId] = newVesting;
         //slither-disable-next-line uninitialized-state-variables
-        IAtomicSwapBase.Release[] storage _releases = releaseInfos[beneficiary][
-            orderId
-        ];
+        IAtomicSwapBase.Release[] storage _releases = releaseInfos[newVestingId];
         for (uint256 i = 0; i < releases.length; i++) {
             _releases.push(releases[i]);
         }
+
+        
         emit NewVesting(
-            VestingInfo(newVesting, releases, beneficiary, orderId)
+            VestingInfo(newVesting, releases, msg.sender, orderId)
```

Comment:
I am thinking if we need use `token_id` instead of `msg.sender`

Reply:
let me remove this variable. currently, buyer and tokenId have no special meaning. 
tokenId can be calculated by BigInt(orderId). 
so I guess we can remove this varaible fully